.github/actions/cypress-e2e-testing/action.yml

```diff
@@ -36,8 +39,13 @@ runs:
           find: '/'
           replace: '-'
 
+    - name: "Start Openid container"
+      if: ${{ contains(steps.formatFeatureFilePath.outputs.value, 'OpenID') || contains(steps.formatFeatureFilePath.outputs.value, 'Web-SSO') }}
+      run: docker run -p 8080:8080 -d -v "/var/run/docker.sock:/var/run/docker.sock" --name e2e-tests-openid-${{ inputs.module }} centreon.jfrog.io/docker/openid:${{ inputs.image_openid_version }}
```

Comment:
It could be a good idea to run this container while developing tests. I could run the container like the Centreon container

Reply:
@Thebarda So do i added it in the before function in tests that require it ? if so then yes but this "inputs.image_openid_version" changes with time how can we get it automaticaly from cypress ?

Reply:
I think you have to use this one: centreon.jfrog.io/docker/openid:23.04